package.json

```diff
@@ -59,6 +59,7 @@
 	 "@financial-times/origami-service-makefile": "7.0.3",
 	 "cheerio": "1.0.0-rc.3",
 	 "dotenv": "8.2.0",
+	 "esbuild": "^0.12.15",
```

Comment:
non-blocking: esbuild isn't used in production, it can be moved dev-dependencies and then Heroku will not include it in the production build